src/universal/types/auth.js

```diff
@@ -4,6 +4,23 @@
  * @flow
  */
 
+export type AuthRole =
+  | 'su';
+
+/**
+ * A deserialized authentication token.
+ */
+export type AuthToken = {
```

Comment:
consider using AuthToken from schema.flow

Reply:
No such definition exists

Reply:
in `rootSchema#types` we can add the AuthToken type & then use it.